gwpy/frequencyseries/frequencyseries.py

```diff
@@ -269,6 +270,37 @@ def zpk(self, zeros, poles, gain, analog=True):
         """
         return self.filter(zeros, poles, gain, analog=analog)
 
+    def interpolate(self, df, left=0, right=0):
+        """Interpolate this `FrequencySeries` to a new resolution.
+
+        Parameters
+        ----------
+        df : `float`
+            desired frequency resolution of the interpolated `FrequencySeries`,
+            in Hz
```

Comment:
can you leave a blank line between parameter definitions in the docstring, it makes things a little easier to read IMO

Reply:
Sure, I'll make this change as well